### PR TITLE
[AIOE-628] update all codelabs to work with extension registry

### PR DESCRIPTION
**Pull request by Yu1986**
## Description

Update all the codelabs in https://www.adobe.io/project-firefly/docs/resources/ to follow the extension registry
https://www.adobe.io/project-firefly/docs/guides/extensions/extension_migration_guide/

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Yu1986** [comment]
@duynguyen  @icaraps  please take a look at the PR for https://jira.corp.adobe.com/browse/AIOE-628

**Review by icaraps — APPROVED** [review]
LGTM, just a few minor change requests 👍

**Comment by Yu1986** [comment]
fix the typo and merge

File: src/pages/resources/asset-compute-worker-ps-api/lesson3.md [hunk 1]
```diff
@@ -67,23 +67,27 @@ exports.main = worker(async (source, rendition, params) => {
 });
 ```
 
-Because the SDK requires `apiKey` to be passed in the input params, update your manifest to include it.
+Because the SDK requires `apiKey` to be passed in the input params, update your `app.config.yaml` to include it.
```
Comment (duynguyen) [inline src/pages/resources/asset-compute-worker-ps-api/lesson3.md:70]: For asset compute, the manifest info now moves to ext.config.yaml inside the extension folder, not app.

File: src/pages/resources/custom-asset-compute-worker/lesson3.md [hunk 2]
```diff
@@ -45,29 +45,34 @@ IMGIX_SECURE_TOKEN=imgx-token
 IMGIX_DOMAIN=your-subdomain.imgix.net
 ```
 
-Edit `manifest.yml` file and add `inputs` object, as shown below. This file describes IO Runtime action to be deployed.
+Edit `app.config.yaml` file and add `inputs` object, as shown below. This file describes IO Runtime action to be deployed.
```
Comment (duynguyen) [inline src/pages/resources/custom-asset-compute-worker/lesson3.md:48]: For asset compute, the manifest info now moves to ext.config.yaml inside the extension folder, not app.

File: src/pages/resources/customer-dashboard/lesson2.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ Within the newly created app, you have seen the `.env` file which contains your
 
 Firstly, `package.json` is the [crucial part](https://docs.npmjs.com/creating-a-package-json-file) of almost every NodeJS project. It contains the list of dependencies, version, reproducible builds, etc.
 
-Then `manifest.yml` is the cockpit of your Firefly app backend. It lists the declaration of serverless actions including name, source files, runtime kind, default params, annotations, and so on. You can find the grammar of writing manifest [here](https://github.com/apache/openwhisk-wskdeploy/blob/master/docs/programming_guide.md#wskdeploy-utility-by-example).
+Then `app.config.yaml` is the cockpit of your Firefly app backend. It lists the declaration of serverless actions including name, source files, runtime kind, default params, annotations, and so on. You can find the grammar of writing manifest [here](https://github.com/apache/openwhisk-wskdeploy/blob/master/docs/programming_guide.md#wskdeploy-utility-by-example).
```
Comment (duynguyen) [inline src/pages/resources/customer-dashboard/lesson2.md:17]: For ExC app, the manifest info now moves to ext.config.yaml inside the extension folder, not app.

**Review by duynguyen — CHANGES_REQUESTED** [review]
Thanks @Yu1986 .
For the "no extension" codelabs, that's all good.
For the extension ones, the manifest goes to ext.config..yaml, inside the extension folder. The app.config.yaml only contains reference of the ext.

**Comment by Yu1986** [comment]
@duynguyen fix all your input, please take a look again

**Comment by duynguyen** [comment]
Thanks @Yu1986 . There're still some minor things to change but I've merged for now and will update directly.

**Pull request merged**